readme.md

```diff
@@ -60,6 +60,13 @@ Content about: Mathematics, Programming languages \
 Featured playlists: `Learn Algebra`, `C++ Tutorial`, `C# Tutorial`, `Java Video Tutorial`, `a lot of diverse topics`. \
 <br/>
 
+[<img align="left" height="94px" width="94px" alt="Alex Lee's channel's avatar" src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRvZUQVPqQ8ipj4pv9rqVt9HCKTzrRsyYdZLcQ_7Ue8U_MV_SYT%3Ahttps%3A%2F%2Fyt3.ggpht.com%2Fa%2FAATXAJznc40hgjUtfJdS_KNadxC_6isyQON0Hw54tGJ92w%3Ds900-c-k-c0x00ffffff-no-rj&usqp=CAU"/>](https://www.youtube.com/channel/UC_fFL5jgoCOrwAVoM_fBYwA/featured)
```

Comment:
```suggestion
[<img align="left" height="94px" width="94px" alt="Alex Lee's channel's avatar" src="https://yt3.ggpht.com/a/AATXAJznc40hgjUtfJdS_KNadxC_6isyQON0Hw54tGJ92w=s100-c-k-c0xffffffff-no-rj-mo"/>](https://www.youtube.com/channel/UC_fFL5jgoCOrwAVoM_fBYwA)
```
Change the link of the image. The link you put is good, but it's not the same you can copy & paste right-clicking on the image, as wrote in the [guidelines](https://github.com/JoseDeFreitas/awesome-youtubers/blob/main/contributing.md).
The channel link shouldn't have any parameter. "featured" is a parameter. I know that parameter redirects you to the main page of the channel, but it's preferable not to use any parameter to keep the same as the other YouTuebers.